EventKit/eventkit-cloud
# fixes mapproxy and find in zip

# Description of Improvement

Fixes unable to save data provider on admin page with no configuration.
Adds back additional integration tests.
Uses updated version of gdalutils which fixes issues with geospatial boundaries and different projections.
Improves mapproxy code to include some types and fixes projection issues.
Prevents styles from causing pipline to fail when converting styles.
Fixes reprojection issues with OSM.
Removes missing tables from osm gpkg_contents table.
Fixes creating styles when using a single layer from an arcgis feature service.
Fixes issue cleaning up load data script. 
Adds support for ogcapi zip files to have multiple files.

## How to test

Create an export using OSM data, osm tiles, and an ogcapi export with different projections.  Ensure that the points, lines, and polygons table doesn't show up in qgis.  Ensure that the raster datasets appear correctly in qgis.  

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
